docs/advanced/styleguide.rst

```diff
@@ -0,0 +1,192 @@
+.. _guide-style-guide:
+
+=============================
+The Guide Style Guide - DRAFT
+=============================
+
+.. note:: This document is attributed to Kenneth Reitz's excellent Python
```

Comment:
I'm -0 on including this by content. Can we get away with linking to Kenneth Reitz's rST style guide, and then (optionally) stating any differences?

Also, we should clarify somewhere if this is a style guide for the docs, or a style guide for the Python code.